File: app/src/main/java/com/example/fixitnopanic/SharedPrefsHelper.java
package com.example.fixitnopanic;

import android.content.Context;
import android.content.SharedPreferences;

public class SharedPrefsHelper {
    private final SharedPreferences prefs;

    public SharedPrefsHelper(Context context) {
        prefs = context.getSharedPreferences("AppPrefs", Context.MODE_PRIVATE);
    }

    public void setCurrentScreen(String screen) {
        prefs.edit().putString("current_screen", screen).apply();
    }

    public String getCurrentScreen() {
        return prefs.getString("current_screen", "main");
    }
}

File: app/src/main/java/com/example/fixitnopanic/SharedPrefsHelper.kt
package com.example.fixitnopanic

import android.content.Context
import android.content.SharedPreferences

class SharedPrefsHelper(context: Context) {
    private val prefs: SharedPreferences = context.getSharedPreferences("AppPrefs", Context.MODE_PRIVATE)

    fun setCurrentScreen(screen: String) {
        prefs.edit().putString("current_screen", screen).apply()
    }

    fun getCurrentScreen(): String {
        return prefs.getString("current_screen", "main") ?: "main"
    }
}